pkg/sql/ttl/ttljob/ttljob_processor.go

```diff
@@ -387,40 +387,44 @@ func (t *ttlProcessor) runTTLOnSpan(
 
 // keyToDatums translates a RKey on a span for a table to the appropriate datums.
 func keyToDatums(
-	key roachpb.RKey, codec keys.SQLCodec, pkTypes []*types.T, alloc *tree.DatumAlloc,
+	rKey roachpb.RKey, codec keys.SQLCodec, pkTypes []*types.T, alloc *tree.DatumAlloc,
```

Comment:
Also rename the confusingly named `key` (which is a RKey) to `rKey` and `rKey` (which is a Key) to `key` to match their types.